go.mod

```diff
@@ -516,7 +518,7 @@ replace (
 	modernc.org/strutil => modernc.org/strutil v1.0.0
 	modernc.org/xc => modernc.org/xc v1.0.0
 	rsc.io/pdf => rsc.io/pdf v0.1.1
-	sigs.k8s.io/apiserver-network-proxy/konnectivity-client => sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.23
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client => sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.25
```

Comment:
no changes in vendor... this diff didn't include any client files, I guess?

Reply:
Went and explicitly checked. The last konnectivity-client fix appears to be August 2. However I like to just keep these 3 pieces in sync when we update.